packages/sample-app/sample_app/openai_vision_base64_example.py

```diff
@@ -0,0 +1,84 @@
+import base64
+import os
+from openai import OpenAI
+from traceloop.sdk import Traceloop
+from traceloop.sdk.decorators import workflow, task
+
+
+api_key = os.environ.get("OPENAI_API_KEY")
+
+
+def encode_image(image_path):
+    with open(image_path, "rb") as image_file:
+        return base64.b64encode(image_file.read()).decode('utf-8')
+
+
+image_path = "/Users/galklm/Downloads/elephant.jpeg"
```

Comment:
nit: maybe add an image together with the sample app and point to it here so people can actually run the sample app and see it in action?

Reply:
will do